CHANGELOG.md

```diff
@@ -0,0 +1,162 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Highlights
+
+#### Basic X11 backend for druid-shell. ([#599])
+
+[@crsaracco] has implemented basic support to run druid on bare-metal X11!
+
+While still incomplete this lays the foundation for running druid on Linux without relying on GTK.
+
+#### Mostly complete Wasm backend for druid-shell. ([#759])
+
+[@elrnv] continued the work of [@tedsta] and implemented a mostly complete Wasm backend and enabled all druid example to [run in the browser](https://elrnv.github.io/druid-wasm-examples/).
+
+While some features like the clipboard, menus or file dialogs are not yet available, all fundamental features are there.
+
+### Added
+
+- `TextBox` can receive `EditAction` commands. ([#814] by [@cmyr])
+- `Split::min_splitter_area(f64)` to add padding around the split bar. ([#738] by [@xStorm])
+- Published `druid::text` module. ([#816] by [@cmyr])
+- Basic X11 backend for druid-shell. ([#599] by [@crsaracco])
+- `InternalEvent::MouseLeave` signalling the cursor leaved the window. ([#821] by [@teddemunnik])
+- `children_changed` now always includes layout and paint request. ([#839] by [@xStorm])
+- Mostly complete Wasm backend for druid-shell. ([#759] by [@elrnv])
+- `UpdateCtx::submit_command`. ([#855] by [@cmyr])
+- `request_paint_rect` for partial invalidation. ([#817] by [@jneem])
+- Window title can be any `LabelText` (such as a simple `String`). ([#869] by [@cmyr])
+- `Label::with_font` and `set_font`. ([#785] by [@thecodewarrior])
+
+### Changed
```

Comment:
There's another significant change:

```markdown
- Global `Application` associated functions are now instance methods instead, e.g. `Application::global().quit()` instead of the old `Application::quit()`. ([#763] by [@xStrom])
```